src/ui/components/ConfirmButton/index.js

```diff
@@ -0,0 +1,122 @@
+/* @flow */
+import makeClassName from 'classnames';
+import * as React from 'react';
+import { compose } from 'redux';
+
+import translate from 'core/i18n/translate';
+import Button from 'ui/components/Button';
+import type { I18nType } from 'core/types/i18n';
+
+import './styles.scss';
+
+
+type Props = {|
+  buttonType?: string,
```

Comment:
Why is this marked as optional when it has a default value? Won't it always have a value of at least `neutral` because of the defaults?

Reply:
Interesting, so I was about to write: it is optional as in "can be omitted when using the component" and not as in "nullable", but I checked the Flow documentation and things have changed apparently:

> Note: You don’t need to make foo nullable in your Props type. Flow will make sure that foo is optional if you have a default prop for foo.

https://flow.org/en/docs/react/components/

Removing the `?` now :)